inst/tests/tests.Rraw

```diff
@@ -16910,3 +16910,14 @@ dt = data.table(time = 1:8, v = INT(5,7,6,1,8,4,2,3))
 dt[time == 2L, v := 2L]
 dt[time == 7L, v := 7L]
 test(2140, dt[dt, on=.(time>time, v>v), .N, by=.EACHI], data.table(time=1:8, v=INT(5,2,6,1,8,4,7,3), N=INT(3,5,2,4,0,1,0,0)))
+
+# repeat of test 450 for #4402
+DT = data.table(a=1:3,b=4:6)
+test(2141.1, rbind(DT,list(c=4L,a=7L)), error="Column 1 ['c'] of item 2 is missing in item 1")
+old = Sys.getenv("LANGUAGE")
+Sys.setenv(LANGUAGE="zh_CN")
```

Comment:
I tried to trigger the #4402 error in the [`snprintf-windows-test`](https://github.com/Rdatatable/data.table/tree/snprintf-windows-test) branch earlier but wasn't getting anywhere, you can see the 5 commits there used various versions of this, I may have been doing something wrong